spec/default.mspec

```diff
@@ -31,6 +31,16 @@ class MSpecScript
     ]
   end
 
+  # Disable to run for bundled gems in test-spec
+  set :bundled_gems, (File.readlines("#{srcdir}/gems/bundled_gems").map do |line|
+    next if /^\s*(?:#|$)/ =~ line
+    "#{srcdir}/spec/ruby/library/" + line.split.first
+  end.compact)
+  set :stdlibs, Dir.glob("#{srcdir}/spec/ruby/library/*")
+  set :library, get(:stdlibs).to_a - get(:bundled_gems).to_a
```

Comment:
FYI another way to do this is https://github.com/oracle/truffleruby/blob/93693825fb81c8ec875883bec587f11a49462a1f/spec/truffleruby.mspec#L77-L82.
But this is fine as well.